mlperf_logging/rcp_checker/1.0.0/rcps.json

```diff
@@ -0,0 +1,464 @@
+{
```

Comment:
Probably better to split this file into one file per model under `1.0.0/rcps/`?

Reply:
That's a good idea, it lets people navigate into the RCPs easier. I will try to add it in one of my subsequent PRs if that is fine with you.